packages/synapse-react-client/src/components/styled/StyledTableContainer.tsx

```diff
@@ -43,6 +43,14 @@ export const StyledTableContainer: StyledComponent<StyledTableContainerProps> =
         : {
             backgroundColor: theme.palette.grey[100],
           },
+      ['tr']: {
+        backgroundColor: theme.palette.background.default,
+      },
+      ['tr:nth-of-type(even)']: noStripedRows
+        ? undefined
+        : {
+            backgroundColor: theme.palette.grey[100],
+          },
```

Comment:
Explicitly set the background of the table rows, or else the pinned columns have transparent backgrounds

Reply:
The opacity applies to the entire column including this background (at least on Firefox), so you can still see through it!